pkg/gateway/lb/handler/grpc/handler.go

```diff
@@ -105,7 +109,7 @@ func (s *server) Exists(ctx context.Context, meta *payload.Object_ID) (id *paylo
 				if sspan != nil {
 					sspan.SetStatus(trace.FromGRPCStatus(st.Code(), msg))
 				}
-				return nil
+				return err
```

Comment:
I'm not sure but I think it should not return an error if it is a NotFound error.